src/Umbraco.Core/ObjectExtensions.cs

```diff
@@ -212,7 +212,7 @@ public static Attempt<object> TryConvertTo(this object input, Type target)
                 var convertible2 = input as IConvertible;
                 if (convertible2 != null)
                 {
-                    return Attempt.Succeed(Convert.ChangeType(convertible2, target));
+                    return Attempt.Succeed(Convert.ChangeType(convertible2, Nullable.GetUnderlyingType(target) ?? target));

```

Comment:
I assume that `Nullable.GetUnderlyingType()` is considered expensive, as we have method `GetCachedGenericNullableType()` which is used above and caches the underlying type. Might be good to reuse it here too.

Reply:
Hmm, alright - I'll see if we can use that instead.

Reply:
With the other change made, I don't think this line is required. I've changed it back to what it was before and all tests pass. I think with the new check above `if (target.IsGenericType && GetCachedGenericNullableType(target) != null)` this is not needed?